src/site/markdown/server/manage-cli.md

```diff
@@ -0,0 +1,264 @@
+```
+
+He can list the mailboxes of a specific user:
+
+```
+{cli} ListUserMailboxes user@domain.tld
+```
+
+And finally can create a specific mailbox:
+
+```
+{cli} CreateMailbox #private user@domain.tld INBO.newFolder
+```
+
+## Managing mappings
+
+A mapping is a recipient rewritting rule. There is several kind of rewritting rules:
+
+ - address mapping: rewritte a given mail address into an other one.
+ - regex mapping.
+
+You can manage address mapping like this:
+
+```
+{cli} AddAddressMapping redirected domain.new user@domain.tld
+{cli} RemoveAddressMapping redirected domain.new user@domain.tld
+```
+
+You can manage regex mapping like this:
+
+```
+{cli} AddRegexMapping redirected domain.new .*@domain.tld
+{cli} RemoveRegexMapping redirected domain.new .*@domain.tld
+```
+
+You can view mapping for a mail address:
+
+```
+{cli} ListUserDomainMappings user domain.tld
+```
+
+And all mappings defined on the server:
+
+```
+{cli} ListMappings
+```
+
+## Manage quotas
+
+Quotas are limitations on a group of mailboxes. They can limit the **size** or the **messages count** in a group of mailboxes.
+
+James groups by defaults mailboxes by user (but it can be overridden), and labels each groups with a quotaroot.
+
+To get the quotaroot a given mailbox belongs to:
+
+```
+{cli} GetQuotaroot #private user@domain.tld INBOX
+```
+
+Then you can get the specific quotaroot limitations.
+
+For the number of messages:
+
+```
+{cli} GetMessageCountQuota quotaroot
+```
+
+And for the storage space available:
+
+```
+{cli} GetStorageQuota quotaroot
+```
+
+You see the maximum allowed for these values:
+
+For the number of messages:
+
+```
+{cli} GetMaxMessageCountQuota quotaroot
+```
+
+And for the storage space available:
+
+```
+{cli} GetMaxStorageQuota quotaroot
+```
+
+You can also specify maximum for these values.
+
+
+For the number of messages:
+
+```
+{cli} SetMaxMessageCountQuota quotaroot value
+```
+
+And for the storage space available:
+
+```
+{cli} SetMaxStorageQuota quotaroot value
+```
+
+With value being an integer. Please note the use of units for storage (K, M, G).
+
+Moreover, James allows to specify defaults maximum values, at the server level. Note: syntax is similar to what was exposed previously.
+
+```
+{cli} SetDefaultMaxMessageCountQuota value
+{cli} GetDefaultMaxMessageCountQuota
+{cli} SetDefaultMaxStorageQuota value
+{cli} GetDefaultMaxStorageQuota
+```
+
+## Re-indexing
+
+James allow you to index your data in an indexer, for making search faster. Both ElasticSearch and Lucene are supported.
+
+For some reasons, you might want to re-index your documents (inconsistencies across datastore, migrations).
+
+To reindex all mail of all mailboxes of all users, type:
+
+```
+{cli} ReindexAll
+```
+
+And for a precise mailbox:
+
+```
+{cli} Reindex #private user@domain.tld INBOX
+```
+
+## Sieve scripts quota
+
+James allow you to configure a Sieve mailet, and stores Sieve scripts. You can update them via the ManageSieve protocol,
```

Comment:
You could start with a less technical approach : James implements Sieve email filtering (RFC xxx). To this end, it provides a mailet, etc